docs/deploy-scl3.rst

```diff
@@ -0,0 +1,208 @@
+=================
+Deploying to SCL3
+=================
+
+MDN is served from SLC3, a data center in Santa Clara, California.  It is
+deployed on several persistent virtual machines, managed by Mozilla WebOps.
+Deploying new code to SCL3 takes several steps.  It takes a couple of hours to
+deploy, and the deployer is responsible for emergency issues for the next 24
+hours. This means that there are 1-3 deployments per week, usually between 7 AM
+and 1 PM Pacific, Monday through Thursday.
+
+.. Note::
+
+   This page describes deployment performed by MDN staff. It requires
+   additional setup and permissions not described here. Deployments will
+   not work for non-staff developers, and should not be attempted.
+
+Pre-Deployment: Update Submodules
+---------------------------------
+* Check that the Kuma_ and the KumaScript_ builds are passing.
+
+.. _Kuma: https://travis-ci.org/mozilla/kuma/
+.. _KumaScript: https://travis-ci.org/mdn/kumascript
+
+* Create a pre-push branch from master::
+
+    git remote -v | grep origin
+    # origin  git@github.com:mozilla/kuma.git (fetch)
+    # origin  git@github.com:mozilla/kuma.git (push)
+    git fetch origin
+    git checkout -b pre-push-2017-08-08 origin/master
+
+* Update the submodules::
+
+    git submodule update --init
```

Comment:
Interesting. I haven't used ``--remote`` or ``--branch`` in the past, so I'll have to play around with it and see if it does The Right Thing 👀